src/middlewared/middlewared/client/client.py

```diff
@@ -47,6 +49,20 @@ def wait(self, timeout=None):
 
 CALL_TIMEOUT = int(os.environ.get('CALL_TIMEOUT', 60))
 
+class ReservFDError(Exception):
+    """
+    This class is used as a custom exception raised in `get_reserved_portfd`.
```

Comment:
Too verbose comment that involves talking about unrelated subsystems (alerts). The best place to explain alert-related part is in the alert plugin code, where you catch this exception and skip it.